http.api/src/main/java/ru/tinkoff/qa/neptune/http/api/CommonBodyPublishers.java

```diff
@@ -125,4 +142,161 @@ private CommonBodyPublishers() {
+     * performed by Jackson
+     *
+     * @param t is an object to be serialized to json string
+     * @param mapper is an object mapper
+     * @param charset  of a resulted request body
+     * @param <T> is a type of the object to be serialized
+     * @return a BodyPublisher
+     */
+    public static  <T> HttpRequest.BodyPublisher serializedStringBody(T t, ObjectMapper mapper, Charset charset) {
+        checkArgument(nonNull(t), "Object to be serialized should not be a null value");
+        checkArgument(nonNull(mapper), "Object mapper should not be a null value");
+        checkArgument(nonNull(charset), "Char set should not be a null value");
+        try {
+            return ofString(mapper.writeValueAsString(t), charset);
+        } catch (JsonProcessingException e) {
+           throw new RuntimeException(e);
+        }
+    }
+
+    /**
+     * Serializes given object to json/xml string for the using it as a request body. The serializing is
+     * performed by Jackson
+     *
+     * @param t is an object to be serialized to json string
+     * @param mapper is an object mapper
+     * @param <T> is a type of the object to be serialized
+     * @return a BodyPublisher
+     */
+    public static <T> HttpRequest.BodyPublisher serializedStringBody(T t, ObjectMapper mapper) {
+        return serializedStringBody(t, mapper, UTF_8);
+    }
+
+    /**
+     * Transforms given xml/html document to string body of a request.
+     *
+     * @param document is xml/html document to be used
+     * @param transformer is a transformer of the document to string value
+     * @param charset of a resulted request body
+     * @return a BodyPublisher
+     */
+    public static  HttpRequest.BodyPublisher documentStringBody(Document document,
+                                                                Transformer transformer,
+                                                                Charset charset) {
+        checkArgument(nonNull(document), "Document should not be a null value");
+        checkArgument(nonNull(transformer), "Transformer should not be a null value");
+        checkArgument(nonNull(charset), "Char set should not be a null value");
+
+        DOMSource domSource = new DOMSource(document);
+
+        var sw = new StringWriter();
+        StreamResult sr = new StreamResult(sw);
+        try {
+            transformer.transform(domSource, sr);
+            return ofString(sw.toString(), charset);
+        } catch (TransformerException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    /**
+     * Transforms given xml/html document to string body of a request.
+     *
+     * @param document is xml/html document to be used
+     * @param transformer is a transformer of the document to string value
+     * @return a BodyPublisher
+     */
+    public static HttpRequest.BodyPublisher documentStringBody(Document document, Transformer transformer) {
```

Comment:
+ (org.w3c.dom.Document document, Transformer transformer)

Using explicit arguments class definitions is better to avoid confusions. Look at Document (org.jsoup.nodes.Document) class used below.